NEWS

```diff
@@ -1,3 +1,15 @@
+11-June-2018 IGNITION v0.26.0
+
+  Features:
+
+    - Expand partitioning support to support filesystem reuse like semantics
```

Comment:
Rather than comparing the feature to a different feature, can we just say what it does?